src/modules/auth/index.js

```diff
@@ -280,14 +280,27 @@ export default class AuthModule extends Module {
 			if (that.App.Store != null) {
 				that.App.Store.dispatch({ type: types.AUTH_SESSION_EXPIRATION, sessionExpired: true });
 				// Disable buttons and pagination in the whole screen
-				[...document.querySelectorAll('[class^="btn"]:not(.alert-button), [class*=" btn"]:not(.alert-button), .page-item')].forEach(i => {
+				[...document.querySelectorAll('[class^="btn"]:not(.alert-button), [class*=" btn"]:not(.alert-button), .btn-group a, .page-item, input, select')].forEach(i => {
 					i.classList.add("disabled");
 					i.setAttribute("disabled", "");
 				});
 				// Disable link without class "nav-link" in the whole screen
 				[...document.querySelectorAll('a:not(.nav-link)')].forEach(i => {
 					i.classList.add("disabled-link");
 				});
+				// Disabling the ability to click on parts of widgets
+				[...document.querySelectorAll('.recharts-surface')].forEach(i => {
```

Comment:
@aringocode Same here, what if we decide not to use recharts?

Reply:
I agree that these settings are specific to certain libraries. I deleted these parts, so now if the user wants to click on them, the page will reload)

Reply:
https://github.com/TeskaLabs/asab-webui/assets/65866093/1cea46ec-0bed-4ed1-99d3-72af25c38a98